### DISCUSSION TITLE
OneDrive v2.4.24: unrecognized option --operation_timeout=60

### DISCUSSION THREAD

**Discussion by jrpelegrina** [abraunegg/onedrive#2432] [General]
Hi

I am testing the new version of the client v2.4.24 installed via deb from openSuse.

I have detected that if, for example, I execute the following command**_: onedrive --display-sync-status --verbose --dry-run --operation_timeout=60_**, its execution fails, showing the following error: Unrecognized option --operation_timeout=60

With version v2.4.23 it worked without problems... is it a bug or simply the new version no longer allows this type of parameter?

All the best

**Comment by abraunegg** [comment]
@jrpelegrina 
> --operation_timeout=60**

This is now a 'config' file option only. You must configure this via the config file.

**Reply by jrpelegrina** [reply]
ok
 I'll take it into account
Thank you very much for the speed in the response.
I take this opportunity to congratulate you on the work you are doing.
All the best

**Reply by JKLeckr** [reply]
Just out of curiosity, was there a specific reason why operation-timeout was removed?

**Reply by abraunegg** [reply]
Because HTTP session timeout fix from v2.5.x was backported, and this option became obsolete because of this.
